weblate_web/templates/donate/form.html

```diff
@@ -67,7 +67,7 @@ <h1 class="section-title min-m">{% trans "Your donation" %}</h1>
                 </form>
             </div>
             <p class="small">
-                {% trans "Please note—a full invoice for your donation will be issued immediatelly after payment." %}
+                {% trans "Please note—a full invoice for your donation is issued immediatelly after payment." %}
```

Comment:
Shouldn't it be "after the payment"?

Reply:
This is just the general, like "when payment occurs"
After watering, the lawn will be watered. The watering of the lawn is the action.